add doc on how to use context-tags (#6567)
getsentry/sentry-docs: src/platforms/java/guides/jul/usage/advanced-usage.mdx

  }
}
```

## Context Tags

> Since Sentry 6.0.0, it is possible to define a list of `MDC tags` that will show up as `Tags` in the Sentry UI. `MDC Tags` not in this list are shown as `Context Data` in the UI, just like before.
> To define such tags add a `context-tags` entry with the tags as comma seperated items to the `sentry.properties` file.
> 
> ```properties {tabTitle:Properties}
> dsn=___PUBLIC_DSN___
> context-tags=userId,requestId
> ```
> 
> When calling `MDC.put("userId", "myUserId")` with above configuration in place, the `userId` will show up as a `tag` on the event in the Sentry UI.

Reviewer: ```suggestion
Starting with Sentry version 6.0.0 and up, you can define a list of `MDC tags` that will be viewable as `Tags` in the Sentry UI. `MDC Tags` not in this list will be viewable as `Context Data`. To define tags, add a `context-tags` entry with the tags as comma-separated items to the `sentry.properties` file.

```properties {tabTitle:Properties}
dsn=___PUBLIC_DSN___
context-tags=userId,requestId
```

When you call `MDC.put("userId", "myUserId")` with the above configuration in place, the `userId` will show up as a `tag` on the event in the Sentry UI.
```
